java-checks/src/main/java/org/sonar/java/checks/helpers/QuickFixHelper.java

```diff
@@ -118,4 +136,99 @@ public static String contentForTree(Tree tree, JavaFileScannerContext context) {
 
     return sb.toString();
   }
+
+  public static Optional<JavaTextEdit> newImportEdit(String requiredType, Tree base) {
```

Comment:
What does `base` means in this context?

Reply:
It was to be used as "any base tree to start from", but in fact, we don't need it, we can simply rely on the context, which has an access to the CompilationUnitTree.